nntrainer/models/neuralnet.cpp

```diff
@@ -160,7 +166,8 @@ int NeuralNetwork::compile() {
   }
 
   bool memory_swap = std::get<props::MemorySwap>(model_flex_props);
-	const std::string memory_swap_path = std::get<props::MemorySwapPath>(model_flex_props);
+  const std::string memory_swap_path =
+    std::get<props::MemorySwapPath>(model_flex_props);
```

Comment:
I think you might need to modify nntrainer.ini file as well to set the default value. (May be add meson option to set path)

Reply:
Thanks.
swap properties are added to nntrainer.ini
And, I found meson config to support platform differences.
```
# Install conf
configure_file(
  input: 'nntrainer.ini.in',
  output: 'nntrainer.ini',
  install_dir: nntrainer_confdir,
  configuration: nntrainer_conf
)
```
`nntrainer_confdir` is differ for each platform ('android' or others)

If it needs any other configurations, please let me know.